articles/cosmos-db/sql/sql-query-is-number.md

```diff
@@ -37,17 +37,18 @@ IS_NUMBER(<expr>)
 SELECT   
     IS_NUMBER(true) AS isNum1,   
     IS_NUMBER(1) AS isNum2,  
-    IS_NUMBER("value") AS isNum3,   
-    IS_NUMBER(null) AS isNum4,  
-    IS_NUMBER({prop: "value"}) AS isNum5,   
-    IS_NUMBER([1, 2, 3]) AS isNum6,  
-    IS_NUMBER({prop: "value"}.prop2) AS isNum7  
+    IS_NUMBER("value") AS isNum3, 
+    IS_NUMBER("12") AS isNum4,
+    IS_NUMBER(null) AS isNum5,  
+    IS_NUMBER({prop: "value"}) AS isNum6,   
+    IS_NUMBER([1, 2, 3]) AS isNum7,  
+    IS_NUMBER({prop: "value"}.prop2) AS isNum8  
```

Comment:
```suggestion
    IS_NUMBER("value") AS isTextStringANumber, 
    IS_NUMBER("1") AS isNumberStringANumber,
    IS_NUMBER(null) AS isNullANumber,  
    IS_NUMBER({prop: "value"}) AS isObjectANumber,   
    IS_NUMBER([1, 2, 3]) AS isArrayANumber,  
    IS_NUMBER({stringProp: "value"}.stringProp) AS isObjectStringPropertyANumber, 
    IS_NUMBER({numberProp: 1}.numberProp) AS isObjectNumberPropertyANumber  
```

I'm going to go ahead and recommend labels here too. BTW, thanks for adding the example of a number as a string. To avoid confusion, let's use the same exact number (``1``) for both the number literal and string examples.

I also added a numeric property example for fun 😸